src/pages/q/[[...all]].tsx

```diff
@@ -211,6 +215,8 @@ export const getServerSideProps: GetServerSideProps = async function ({
     )
     try {
       initialInstructor = await loadInstructor(instructorSlug)
+
+      sanityInstructor = await loadSanityInstructor(instructorSlug)
     } catch (error) {
```

Comment:
When an instructor filter is selected we will load sanity data (if it passes the type guard)